server/odc-service/src/main/java/com/oceanbase/odc/service/task/caller/PodConfig.java

```diff
@@ -59,4 +59,8 @@ public class PodConfig {
     private Long mountDiskSize;
 
     private Long maxNodeCount;
+
+    private Long nodeCpu;
```

Comment:
may double, for example cpu can be 0.5 core

Reply:
done